squad/ci/models.py

```diff
@@ -134,10 +134,20 @@ def __postprocess_testjob__(self, test_job):
                 logger.error("Plugin postprocessing error: " + str(e) + "\n" + traceback.format_exc())
 
     def submit(self, test_job):
-        test_job.job_id = self.get_implementation().submit(test_job)
+        job_id_list = self.get_implementation().submit(test_job)
+        if isinstance(job_id_list, list):
```

Comment:
`self.get_implementation().submit(test_job)` will always return a list of IDs, so this condition will always be true

Reply:
It should but I don't know how to enforce it. Anyway, point taken, I'll update it.

Reply:
It initially looked like one line change but ended with 2 commits and half of the tests updated :)